### PR TITLE
Allow RespondToForm to be loaded in the FormulaeExample app

### PR DESCRIPTION
**Pull request by franzejr**

### CODE REVIEW

File: src/output_lib/components/RespondToForm.js [hunk 1]
```diff
@@ -124,6 +125,27 @@ export default function RespondToForm(props: Props) {
     submitted
   } = props;
 
+  if (formId !== 0 && form.id === 0) {
```
Comment (knewter) [inline src/output_lib/components/RespondToForm.js:128]: this is never true

Comment (franzejr) [reply src/output_lib/components/RespondToForm.js:128]: they are different.

Comment (knewter) [reply src/output_lib/components/RespondToForm.js:128]: my eyes suck? :)

**Comment by franzejr** [comment]
Closing this, because of https://github.com/dailydrip/formulae_react/pull/67

**Pull request closed without merging**